src/sendmail/mail_sender.py

```diff
@@ -0,0 +1,58 @@
+import os
+import mailtrap as mt
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+env = Environment(
+    loader=FileSystemLoader("src/sendmail"),
+    autoescape=select_autoescape()
+)
+
+def send_email(args):
```

Comment:
Can we tighten this up by using
```
def send_email(records: list[BuildingServicesSearchResult]):
```
as the function's signature?

Reply:
Done! :)